conf/pawsey_setonix.config

```diff
@@ -0,0 +1,29 @@
+// Pawsey Setonix nf-core configuration profile
+params {
+  config_profile_description = 'Pawsey Setonix HPC profile provided by nf-core/configs'
+  config_profile_contact = 'Sarah Beecroft (@SarahBeecroft), Georgie Samaha (@georgiesamaha)'
+  config_profile_url = 'https://support.pawsey.org.au/documentation/display/US/Setonix+User+Guide'
+}
+
+// Enable use of Singularity to run containers
+singularity {
+  enabled = true
+  autoMounts = true
+  autoCleanUp = true
+}
+
+// Submit up to 256 concurrent jobs (Setonix work partition max)
+executor {
+  queueSize = 1024
+}
+
+// Define process resource limits
+// See: https://support.pawsey.org.au/documentation/pages/viewpage.action?pageId=121479736#RunningJobsonSetonix-Overview
+process {
+  executor = 'slurm'
+  clusterOptions = "--account=${System.getenv('PAWSEY_PROJECT')}"
+  module = 'singularity/3.11.4-slurm'
+  cache = 'lenient'
+  stageInMode = 'symlink'
+  queue = { task.memory < 238.GB ? 'work' : (task.memory > 230.GB && task.memory <= 980.GB ? 'highmem' : '') }
```

Comment:
Is there then a default queue (for the last condition)

Reply:
Oops no, cleaned up